.github/workflows/integ-test.yml

```diff
@@ -98,6 +98,72 @@ jobs:
           )" >> $GITHUB_OUTPUT
     if: "${{ github.event.inputs.integ_tests_include || github.event.schedule }}"
 
+  examples-matrix:
+    runs-on: [ ubuntu-latest ]
+    container: python:3.10-slim-buster
+    env:
+      DEBIAN_FRONTEND: noninteractive
+      ANSIBLE_COLLECTIONS_PATH: /work-dir
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - run: apt update
+      - run: apt install -y jq
+      - run: mkdir -p $WORKDIR
+      - run: cp -a ./  $WORKDIR
+
+      - id: set-matrix
+        shell: bash
+        run: |-
+          echo "matrix=$(
+            ls -r $WORKDIR/examples | jq -R -s -c 'split("\n")[:-1]'
+          )" >> $GITHUB_OUTPUT
+    if: "${{ github.event.schedule }}"
```

Comment:
I see now - I cannot run this manually (workflow-dispatch).

https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-event-information
https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_dispatch

I think this would already allow manual or scheduled start (if `github.event.inputs.integ_tests_include` is None or "" or [], then `github.event.inputs` is a dict, not None) .
```
    if: "${{ github.event.inputs || github.event.schedule }}"

```